man/man8/zpool.8

```diff
@@ -2867,6 +2867,12 @@ or
 by setting
 .Sy ZFS_VDEV_DEVID_OPT_OUT .
 .El
+.Bl -tag -width "ZFS_COLOR"
+.It Ev ZFS_COLOR
+Use ANSI color in
+.Nm zpool status
+output.
+.El
```

Comment:
Can you move this up after `ZFS_ABORT` to keep the environment variables alphabetically ordered.  Moving` `ZFS_VDEV_DEVID_OPT_OUT` up wouldn't be a bad thing either.